health-api/src/main/java/com/netflix/runtime/health/api/IndicatorFilters.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.runtime.health.api;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Static builder of for creating an {@link IndicatorFilter} instance.
+ * 
+ * <pre>
+ *  IndicatorFilters.includes(someIndicatorInstance)
```

Comment:
It does not allow for classes. Not an oversight. There was originally a notion that indicator's should be named, so I kept that around and built the filters around that. To support classes as a parameter I'd have to be able to instantiate the class, which puts constraints on what kind of constructor an indicator could have. Didn't really want that.